deploy-board/deploy_board/settings.py

```diff
@@ -373,4 +373,9 @@
     ENABLE_AMI_AUTO_UPDATE = 1
 
     #Stage Type Info Link
-    STAGE_TYPE_INFO_LINK = os.getenv("STAGE_TYPE_INFO_LINK")
\ No newline at end of file
+    STAGE_TYPE_INFO_LINK = os.getenv("STAGE_TYPE_INFO_LINK")
+
+    #Primary Account
+    AWS_PRIMARY_ACCOUNT = os.getenv("AWS_PRIMARY_ACCOUNT", "998131032990")
+    #Sub Account
+    AWS_SUB_ACCOUNT = os.getenv("AWS_SUB_ACCOUNT", "562567494283")
```

Comment:
can we support multiple sub accounts?

Reply:
yea, this is a very good point. we can support multiple subaccounts in the future. Now, I will add a new option "others" in the dropdown menu.